components/list/list-item-mixin.js

```diff
@@ -90,6 +90,7 @@ export const ListItemMixin = superclass => class extends ListItemDragDropMixin(L
 			}
 			.d2l-list-item-content.d2l-hovering,
 			.d2l-list-item-content.d2l-focusing {
+				--d2l-list-item-content-text-color: var(--d2l-color-celestine);
```

Comment:
Oof. I think changing the variable was my intent. This would leak all over the place, wouldn't it?

Not sure I understand about putting it below in the illustration rule. The intent is to update the color of both the primary text and the illustration icon color.

Reply:
By using a variable it means that consumers can override it higher in the DOM, which as far as I know we don't want them to do. I think it's fine to use the variable and we maintain that it is private. I think an alternative is that we also render the `d2l-hovering` and `d2l-focusing` classes on a the illustration slot or a div that wraps it, thereby allowing celestine to be specified explicitly.